server/bootstrap/routes.js

```diff
@@ -5,16 +5,18 @@ var IndexController = require(process.env.ROOT + '/server/mvc/controllers/IndexC
 var AdminController = require(process.env.ROOT + '/server/mvc/controllers/AdminController.js');
 var Admin_UsersController = require(process.env.ROOT + '/server/mvc/controllers/Admin/UsersController.js');
 var ProfileController = require(process.env.ROOT + '/server/mvc/controllers/ProfileController.js');
+var BattlenetController = require(process.env.ROOT + '/server/mvc/controllers/BattlenetController.js');
 
 module.exports = function (app, logs, passport) {
   app.get('/auth/bnet', passport.authenticate('bnet'));
   app.get('/auth/bnet/callback', passport.authenticate('bnet', {failureRedirect: '/'}), function(req, res){
-    res.redirect('/');
+    res.redirect('/bnet');
```

Comment:
Cool, I like the sound of a loading indicator with user creation in the background.  It takes less than 5 seconds to get all 3 regions' bnet data during development, but I imagine we could put instructions/tips on using the new website, very important news/announcements, or highlights on the loading page.  Then have the user click to continue or something.